log/log.go

```diff
@@ -41,13 +41,30 @@ type Logger interface {
 	Infof(string, ...interface{})
 	Debug(...interface{})
 	Debugf(string, ...interface{})
+	Level() Level
 }
 
 type ctxKey struct{}
 
 // FactoryFunc function type for creating loggers.
 type FactoryFunc func(map[string]interface{}) Logger
 
+var levelPriorities = map[Level]int{
```

Comment:
I think the level should start differently ordered.
Checkou out zerolog

```
const (
	// DebugLevel defines debug log level.
	DebugLevel Level = iota
	// InfoLevel defines info log level.
	InfoLevel
	// WarnLevel defines warn log level.
	WarnLevel
	// ErrorLevel defines error log level.
	ErrorLevel
	// FatalLevel defines fatal log level.
	FatalLevel
	// PanicLevel defines panic log level.
	PanicLevel
	// NoLevel defines an absent log level.
	NoLevel
	// Disabled disables the logger.
	Disabled
)
```
this way you say log only over info which means >= 1